### ISSUE TITLE
Proposal Review 11/7/2018

### ISSUE THREAD

**Issue by BCrawfordScott** [musicpulpite/Feedlet#1]
### MVP List 
- [ ] Flesh out each list item:
   - [ ] Include timeline goal for the completion of each feature.
   - [ ] Include description of the key pieces of each feature. ( See [bluebird](https://github.com/appacademy/bluebird/wiki/mvp-list) for examples)

### Schema
It looks like our MVP list might be a bit outdated as far as entity names are concerned, but I think the tables should look something like this:
- [ ] Add `collections` and `feeds` tables.
   - [ ] `collections` will belong to a user and have a name 
` - [ ] `feeds` will not belong to anything and they will have a column for the url that is the api for that RSS feed.
- [ ] Change `Feed/Sources` to `feed_collections` 
   - [ ] Change the `feed_name` and the `source_name` to `feed_id` and `collection_id`. This will be all the user's collections and their corresponding feeds.
- [x] Update `articles`
   - [ ] remove `source_name` and instead only have `source_id` (articles belong to a source).
   - [ ] add `article_url` for displaying the articles info
- [x] Add `reads` table (this will only hold a user_id and an article_id to represent a read).

Aggregating news sources for an RSS feed is going to involve a lot of third party api calls, and not a lot of data being held in your DB.  Your frontend will do a lot of the work with your redux state to render the data from an article.  Let me know if you have any questions and we'll talk more about this early next week.

**Comment by musicpulpite** [comment]
Take a look at the Feedly website because I think what you are calling **collections** corresponds to what they call **feeds** and what you are calling **feeds** corresponds to what they call **sources**. In other words, the sources correspond to the individual websites (or website sections) and their associated URL's.

Now that I see your terminology, I think it would make a lot more sense to name the individual website links as **feeds** since those are the things being updated by the RSS Feeds, but that looks to be the way that Feedly wants to do it.

**Comment by musicpulpite** [comment]
But when I looked at the MVP list again it does look like App Academy uses the collection terminology as well. If I understand what they are going for, I think it will be a lot easier for me to just follow Feedly's terminology. But let me know.

**Comment by BCrawfordScott** [comment]
Yeah, it seems like there are some naming differences between our listed MVPs and how Feedly is currently organizing its' entities.  I'll pass that to the curriculum team and see if we can get it updated.

Using Feedly's terminology is good with me, as long as the structure/hierarchy of the data stays the same.  Users will have many feeds, sources will have many articles, and feeds will be populated with sources through a joins table.  

Let me know if you have more questions.
